src/tests/JIT/Performance/JIT.performance.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
     <MergedWrapperProjectReference Include="*/**/*.??proj" />
+    <MergedWrapperProjectReference Remove="JittedMethodsCountingTest/HelloWorld/Helloworld.csproj" />
```

Comment:
You can fix this more cleanly by modifying the following line:

https://github.com/dotnet/runtime/blob/7a11cff914fa6fcf00eb59be33306e3826e8b958/src/tests/Directory.Build.targets#L513

as follows:

<pre>
  &lt;PropertyGroup Condition="'$(Language)' == 'C#' and $(_CLRTestNeedsToRun) and ('$(BuildAsStandalone)' == 'true' or '$(RequiresProcessIsolation)' == 'true' or '$(InMergedTestDirectory)' == 'true' or '$(IsMergedTestRunnerAssembly)' == 'true')">
</pre>

Once you do that, it should suffice to state in your ```HelloWorld.csproj``` project that it's ```&lt;CLRTestKind&gt;BuildOnly&lt;/CLRTestKind&gt;``` and it should do the same thing under the covers.